heliopy/data/util.py

```diff
@@ -360,9 +360,10 @@ def load(filename, local_dir, remote_url, guessversion=False,
 
     # Try to load locally
     if _checkdir(local_dir):
-        for f in os.listdir(local_dir):
-            if f == filename or guessversion and (f[:-6] == filename[:-6]):
-                filename = f
+        for f in local_dir.iterdir():
```

Comment:
Does `local_dir.iterdir()` iterate through files in `local_dir`, or does it include the directory path too?